pkg/cmd/step/create/step_create_task.go

```diff
@@ -610,13 +610,14 @@ func (o *StepCreateTaskOptions) createPipelineTaskParams() []pipelineapi.Param {
 func (o *StepCreateTaskOptions) setBuildValues() error {
 	labels := map[string]string{}
 	if o.GitInfo != nil {
-		labels["owner"] = o.GitInfo.Organisation
-		labels["repo"] = o.GitInfo.Name
+		labels[tekton.LabelOwner] = o.GitInfo.Organisation
```

Comment:
This has no relation to the "owner" label [here](https://github.com/jenkins-x/jx/blob/e466371900aa308b36bc3c9dcb76876c976c42ba/pkg/apis/jenkins.io/v1/types_pipeline.go#L12), right? I tried to do a quick search to see if anything relevant was keying directly on "owner" but didn't find anything obvious.

Reply:
Correct. I'm not sure if anything's using the labels on `PipelineActivity`, but this is definitely independent of those.

Reply:
our UI is using `owner` and `repo` labels :)

Reply:
but yeah, all looks good